docker/test/verify.sh

```diff
@@ -40,13 +42,20 @@ if [ -z "${DATE}" ]; then
 fi
 
 imageName="${1}"
+arch="${2:-amd64}"
 
 # Check that the image exists
 if ! imageInfo="$(docker inspect "${imageName}")"; then
```

Comment:
:question: How does inspect know the arch to inspect, especially after you've pulled the second one?

Reply:
see https://github.com/camunda/zeebe/pull/11003#discussion_r1024017807 this depends on the current local state of the image tag/last pull execution.